data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_abilities_class.lst

```diff
@@ -1687,21 +1687,21 @@ Regular Talent	KEY:Rogue Talent ~ Regular Talent		CATEGORY:Special Ability	TYPE:
+# Ability Name			Unique Key							Category of Ability		Type										Define												Description																																																																																																									Ability																				Bonus Ability Pool		Modify VAR																				Source Page
+Class Skills			KEY:Sorcerer ~ Class Skills				CATEGORY:Internal																																																																																																																																				ABILITY:Class Skill|AUTOMATIC|Appraise|Bluff|Craft|Fly|Intimidate|Knowledge (Arcana)|Profession|Spellcraft|Use Magic Device																											SOURCEPAGE:p.70
+Weapon and Armor Proficiency	KEY:Sorcerer ~ Weapon and Armor Proficiency	CATEGORY:Special Ability	TYPE:Class Feature.Sorcerer Class Feature.SpecialQuality														DESC:Sorcerers are proficient with all simple weapons. They are not proficient with any type of armor or shield. Armor interferes with a sorcerer's gestures, which can cause her spells with somatic components to fail.																																																																						ABILITY:Internal|AUTOMATIC|Weapon Prof ~ Auto|Weapon Prof ~ Simple																																				SOURCEPAGE:p.70
+Spells				KEY:Sorcerer ~ Spells					CATEGORY:Special Ability	TYPE:Class Feature.Sorcerer Class Feature.SpecialQuality	DEFINE:Sorcerer_Spells_StatBonus|0	DEFINE:Sorcerer_Spells_SpellDCBonus|0	DESC:A sorcerer casts arcane spells drawn primarily from the sorcerer/wizard spell list. She can cast any spell she knows without preparing it ahead of time. To learn or cast a spell, a sorcerer must have a Charisma score equal to at least 10 + the spell level. The Difficulty Class for a saving throw against a sorcerer's spell is %1 + the spell level.|Sorcerer_Spells_SpellDCBonus																																																																					BONUS:VAR|Sorcerer_Spells_StatBonus|CHA|TYPE=Base	BONUS:VAR|Sorcerer_Spells_SpellDCBonus|10+Sorcerer_Spells_StatBonus	SOURCEPAGE:p.70
+Bloodline				KEY:Sorcerer ~ Standard Bloodline			CATEGORY:Special Ability	TYPE:Class Feature.Sorcerer Class Feature.Bloodline		DEFINE:HasSorcererBloodline|0									DESC:Each sorcerer has a source of magic somewhere in her heritage that grants her spells, bonus feats, an additional class skill, and other special abilities. This source can represent a blood relation or an extreme event involving a creature somewhere in the family's past. For example, a sorcerer might have a dragon as a distant relative or her grandfather might have signed a terrible contract with a devil. Regardless of the source, this influence manifests in a number of ways as the sorcerer gains levels. A sorcerer must pick one bloodline upon taking her first level of sorcerer. Once made, this choice cannot be changed.	ABILITY:Internal|AUTOMATIC|Bloodline Tracker														BONUS:ABILITYPOOL|Bloodline|1	BONUS:VAR|HasSorcererBloodline|1|TYPE=Base														SOURCEPAGE:p.71
```

Comment:
Why not make this a FACT?

Reply:
Because you can't add a value to a FACT. This is used to replace CHA as a bonus to sorcerer abilities, so it can be easily increased in a single place for Sorcerer Affinity Bonus.